### ISSUE TITLE
bug: ionic 4 does not allow pinch to zoom capture on maps like google map ios

### ISSUE THREAD

**Issue by esteban03** [ionic-team/ionic#18748]
# Bug Report

** Ionic version: **
ionic version: 4
cli version: 5.2.1

** Current behavior: **
When doing the gesture in google maps (I'm testing with angular agm-map) it is not captured. It also does not work on maps such as leaflet or mapbox. It is apparently canceled by the fraemwork.

** Expected behavior: **
Ionic captures the gesture with the fingers and to zoom in on the map.

** Steps to play: **
1. Create project with angular.
2. Install google maps, I recommend agm-map for efficiency. https://angular-maps.com/
3. Compile for ios.
4. Try to zoom in and zoom out with your fingers on the map.

Try using data-tap-disabled = "true" but I did not succeed.

**Comment by liamdebeasi** [comment]
Thanks for the issue. Can you provide a repository with the code required to reproduce this issue? I am interested in seeing how you are using your map in an Ionic app.

**Comment by esteban03** [comment]
no problem, in this repository leave an application with google maps and leaflet. The problem of pinch to zoom occurs with both maps.

https://github.com/esteban03/ionic-map-fail-example

**Comment by liamdebeasi** [comment]
Thanks for the follow up. Which device/OS version are you testing this on? I am testing on an iPhone running iOS 12 and pinch to zoom is working fine.

**Comment by esteban03** [comment]
Thank you for answering so quickly. I performed tests on an iphone 8 and XR. In both I had this problem and also occurs in emulators. Does it work with the gmaps and leaflet? It seems very strange to me.

**Comment by liamdebeasi** [comment]
Yeah, it was working on all the different maps I tried. Which version of iOS are you running on these devices?

**Comment by esteban03** [comment]
I am using ios 12. Remove the npm modules and install ionic-webview 4.1.0 to solve a problem with the publication of the app and I realized that the problem was solved when testing it on physical devices. In the emulator it still does not work but although it is annoying it is not a big problem if it can be used in the final device. I really do not know if it was the webview or another module reinstalled but this solved my problem that was causing me great headaches.
